satellite/satellitedb/offers.go

```diff
@@ -118,6 +118,7 @@ func (offers *offers) Redeem(ctx context.Context, oID int) error {
 	return nil
```

Comment:
Should we create a helper function that checks if `num_redeemed` < `redeemable_cap` before executing the query statement?

Reply:
The check is done in the query